### ISSUE TITLE
Investigate Tim's ageutils as a potential dependency

### ISSUE THREAD

**Issue by Bisaloo** [socialcontactdata/contactmatrix#10]
https://timtaylor.github.io/ageutils/index.html

**Comment by TimTaylor** [comment]
This is still quite experimental so best to track the dev docs https://timtaylor.github.io/ageutils/dev/ until it reaches 0.1.0. In case you've already played around have a gander at the [changelog](https://timtaylor.github.io/ageutils/dev/news/index.html) as I've deprecated some things.

I'm hoping to push the current dev version to CRAN as 0.0.5 once blackedder has had a play.

After this the roadmap is roughly:

- Remove the C code (this was from a time when we thought we'd be sharing the header with other compiled code which is no longer the case).
-  I then need to spend some time with Blackedder working out how to best handle groups (dplyr/split lapply type approaches aren't appropriate here).
- Finally need a vignette with more realistic motivating examples / case studies 

Once these three things are complete I'll likely push a 0.1.0 but this isn't likely before October due to other commitments.

**Comment by sbfnk** [comment]
This seems to be on CRAN now with version 0.1.1

**Comment by TimTaylor** [comment]
Yep - on CRAN and the current functionality is stable. It has had a reasonable kicking now and I think we flushed out most of the bugs (predominantly handling oddities when reaggregating rates which took more submissions than I'd like to admit to get correct). Unsure if it will be much help to yourselves but feel free to poke me if useful.
